Doc/Tutorial/chapter_pdb.tex

```diff
@@ -93,24 +93,28 @@ \subsection{Reading an mmCIF file}
 \subsection{Reading files in the MMTF format}
 
 You can use the direct MMTFParser to read a structure from a file:
+%doctest ../Tests lib:mmtf
 \begin{verbatim}
 >>> from Bio.PDB.mmtf import MMTFParser
 >>> structure = MMTFParser.get_structure("PDB/4CUP.mmtf")
 \end{verbatim}
 
 Or you can use the same class to get a structure by it's PDB ID:
+%doctest ../Tests lib:mmtf
 \begin{verbatim}
 >>> structure = MMTFParser.get_structure_from_url("4CUP")
 \end{verbatim}
 
 This gives you a Structure object as if read from a PDB or mmCIF file.
 
-You can also have access to the underlying data using the MMTF module.
+You can also have access to the underlying data using the external MMTF library which Biopython is using internally:
+%doctest ../Tests lib:mmtf
 \begin{verbatim}
 >>> from mmtf import fetch
 >>> decoded_data = fetch("4CUP")
 \end{verbatim}
 For example you can access just the X-coordinate.
+%doctest ../Tests lib:mmtf
```

Comment:
This can't be a doctest as the output of the print is missing. It would probably be too long to use as a doctest.